modules/Bio/EnsEMBL/Compara/PipeConfig/EBI/Metazoa/Lastz_conf.pm

```diff
@@ -0,0 +1,104 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::PipeConfig::Lastz_conf
+
+=head1 SYNOPSIS
+
+    #1. Update ensembl-hive, ensembl and ensembl-compara GIT repositories before each new release
+
+    #3. Check all default_options in PairAligner_conf.pm, especically:
+        release
+        pipeline_db (-host)
+        resource_classes 
+
+    #4. Check all default_options below, especially
+        ref_species (if not homo_sapiens)
+        default_chunks (especially if the reference is not human, since the masking_option_file option will have to be changed)
+        pair_aligner_options
+
+    #5. Run init_pipeline.pl script:
+        Using command line arguments:
+        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::EBI::Metazoa::Lastz_conf --dbname hsap_btau_lastz_64 --password <your password> --mlss_id 534 --pipeline_db -host=compara1 --ref_species homo_sapiens --pipeline_name LASTZ_hs_bt_64 
```

Comment:
```suggestion
                init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::EBI::Metazoa::Lastz_conf -host mysql-ens-compara-prod-X -port XXXX \
            -mlss_id 534 -ref_species homo_sapiens
```
It would be better if you could edit this with the actual parameters you used (except `-host` and `-port`).